.github/workflows/image.yml

```diff
@@ -0,0 +1,27 @@
+name: Building Base Image
+
+on:
+  workflow_dispatch:
```

Comment:
I think we should add this 
push:
    paths:
    - 'test/Containerfile-build'